packages/frontend/tests/integration/components/user-guide-link-test.js

```diff
@@ -0,0 +1,30 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'frontend/tests/helpers';
+import { render, settled } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupIntl } from 'ember-intl/test-support';
+
+module('Integration | Component | user-guide-link', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks, 'en-us');
+
+  test('it renders', async function (assert) {
+    await render(hbs`<UserGuideLink />`);
+
+    assert.dom('.user-guide-link').exists();
+    assert.dom('.user-guide-link svg.fa-circle-question').exists();
+    assert.dom('.user-guide-link svg.fa-circle-question title').hasText('Ilios User Guide');
+
+    this.owner.lookup('service:intl').setLocale('es');
```

Comment:
You don't need to test the translations, but if you want to follow the pattern in https://ember-intl.github.io/ember-intl/docs/guide/testing to set the locale or test the untranslated output.

Reply:
Updated tests to be more in line with convention.